packages/gestalt/src/SearchGuide.css

```diff
@@ -0,0 +1,190 @@
+.searchguide {
+  composes: borderBox minWidth60 from "./Layout.css";
+  composes: noBorder from "./Borders.css";
+  composes: pointer from "./Cursor.css";
+  composes: large from "./Layout.css";
+  composes: paddingX1 from "./boxWhitespace.css";
+  border-radius: var(--rounding-600);
+}
+
+.searchguide a,
+:link,
+:visited,
+:hover,
+:active {
+  text-decoration: none;
+}
+
+.searchguideVr {
+  border-color: transparent;
+  border-radius: var(--rounding-200);
+  box-sizing: border-box;
+  min-height: 48px;
+  padding: var(--space-0) var(--space-0);
+  text-wrap: balance;
+}
+
+.parentSearchGuide {
+  composes: paddingX0 from "./boxWhitespace.css";
+  composes: paddingY0 from "./boxWhitespace.css";
+  background-color: var(--color-transparent);
+}
+
+.childrenDiv {
+  composes: flex from "./Layout.css";
+  composes: xsItemsCenter from "./Layout.css";
+  composes: justifyCenter from "./Layout.css";
+  min-height: 48px;
+}
+
+/* STATES */
+
+.selected {
+  border: 2px solid;
```

Comment:
can we compose classes from Borders.css here?

Reply:
Included in next commit :)
I'll keep the direct references for the VR classes though, to keep things better isolated for after the experiment.